libs/common/src/sql-db-connection/migrations/1734616272101-AddFeeRecordCorrectionTransientFormDataTable.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFeeRecordCorrectionTransientFormDataTable1734616272101 implements MigrationInterface {
+  name = 'AddFeeRecordCorrectionTransientFormDataTable1734616272101';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TABLE "FeeRecordCorrectionTransientFormData" (
+                "userId" nvarchar(255) NOT NULL,
+                "correctionId" int NOT NULL,
+                "formDataSerialized" nvarchar(1000) NOT NULL,
+                "lastUpdatedAt" datetime2 NOT NULL CONSTRAINT "DF_6bf1a666c5c54f26a80a93cb477" DEFAULT getdate(),
```

Comment:
could hardcoding the id not cause issues later on if the id changes for any reason? or if duplicated.  Ignore me if not an issue

Reply:
I don't think it does cause issues, I tagged Christian to double check and he has confirmed this is something we are fine to set ourselves, it's just a name for the constraint